src/app/components/case/tasks/ChangeDueDate/ChangeDueDateForm.tsx

```diff
@@ -20,12 +21,7 @@ const Div = styled.div`
   }
 `
 
-const Input = styled.input`
-  margin: ${ themeSpacing(6) } 0;
-  padding: ${ themeSpacing(2) } ${ themeSpacing(3) };
-`
-
-const formatDate = (dateStr: string) => {
+const formatDate = (dateStr: string | Date) => {
   const date = new Date(dateStr)
   const day = ("0" + date.getDate()).slice(-2)
```

Comment:
Can we do `` `0${ date.getDate() }` ``? :)

Reply:
Sorry, you only mentioned it 3 times before ;-)